### PR TITLE
PROD-2: Added footer, cleared App.css

### PR DESCRIPTION
**Pull request by VitaliiChverenko**

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            <p>+01 678 345 12</p>
+          </div>
+        </div>
+      <div class="ui container">
+        ProDobro 2018. All Rights Reserved
+      </div>
+    </div>
+  )
+}
+export default Footer
```
Comment (Pavliv) [inline src/components/Footer.js]: Please add new line

File: src/components/Navigation.js [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
-export default Navigation;
+  <ul>
+    <li><Link to={ routes.NEWS }>News</Link></li>
+    <li><Link to={ routes.CAMPAIGNS }>News</Link></li>
+    <li><Link to={ routes.CONTACTS }>Contacts</Link></li>
+    <li><Link to={ routes.ABOUT_US }>About us</Link></li>
+    <li><Link to={ routes.SIGN_IN }>Sign in</Link></li>
+    <li><Link to={ routes.SIGN_UP }>Sign up</Link></li>
+  </ul>
+export default Navigation;
```
Comment (Pavliv) [inline src/components/Navigation.js]: and here

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+Fixes # (issue)
+
+
+# Checklist:
+- [ ] PR is approved by 1 team mates
```
Comment (Pavliv) [inline pull_request_template.md]: should be singular

**Review by Pavliv — APPROVED** [review]

File: src/components/Navigation.js [hunk 4]
```diff
@@ -1,16 +1,15 @@
...
-      <li><Link to={routes.NEWS}>News</Link></li>
-      <li><Link to={routes.CONTACTS}>Contacts</Link></li>
-      <li><Link to={routes.ABOUT_US}>About us</Link></li>
-      <li><Link to={routes.SIGN_IN}>Sign in</Link></li>
-      <li><Link to={routes.SIGN_UP}>Sign up</Link></li>
-    </ul>
-  </div>
+  <ul>
+    <li><Link to={ routes.NEWS }>News</Link></li>
+    <li><Link to={ routes.CAMPAIGNS }>News</Link></li>
```
Comment (Pavliv) [inline src/components/Navigation.js]: Should be Campaign?

**Review by Antoniozinchenko — APPROVED** [review]

**Pull request merged**